src/js/components/ServiceSearchFilter.js

```diff
@@ -0,0 +1,65 @@
+import mixin from 'reactjs-mixin';
+/* eslint-disable no-unused-vars */
+import React from 'react';
+/* eslint-enable no-unused-vars */
```

Comment:
Then again – this probably should be a consensus decision. Let's skip it for now.